src/main/java/com/spring/app/controllers/CompanyController.java

```diff
@@ -3,38 +3,36 @@
 import com.spring.app.domain.Company;
 import com.spring.app.repos.CompanyRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 
 import java.util.Map;
 
+/**
+ * CompanyController
+ *
+ * @author lyubov
+ */
 @Controller
-public class CompanyContoller {
+public class CompanyController {
     @Autowired
     private CompanyRepository companyRepository;
 
     @GetMapping("companies")
-    public String main(Map<String, Object> model) {
-        Iterable<Company> messages = companyRepository.findAll();
-
-        model.put("companies", messages);
-
-        return "company";
+    public ResponseEntity<Iterable<Company>> main(Map<String, Object> model) {
```

Comment:
Have you heard smth about DTO - `Data transfer Object`. DO NOT USE entity in REST API!

Reply:
main is not a good name...